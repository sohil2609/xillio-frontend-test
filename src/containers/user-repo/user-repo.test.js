import React from 'react';
import { shallow } from 'enzyme';
import UserRepo from './user-repo.component';

const setUp = (props = {}) => {
	const component = shallow(<UserRepo {...props} />);
	return component;
};

describe('User Repo Component', () => {
	let component;
	beforeEach(() => {
		component = setUp();
	});

	it('Should test snapshot', () => {
		expect(component).toMatchSnapshot();
	});
});
