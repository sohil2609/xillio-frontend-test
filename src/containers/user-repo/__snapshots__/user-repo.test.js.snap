// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Repo Component Should test snapshot 1`] = `ShallowWrapper {}`;
